### ISSUE TITLE
react-onsenui compatibility with onsenui 2.2.0 

### ISSUE THREAD

**Issue by GoaGit** [OnsenUI/OnsenUI#1923]
__Environment__ 

```
[Core]
    onsenui 2.2.0

[Framework]
    react 15.4.1

[Framework binding]
    react-onsenui 1.1.3

[Additional libraries]
    none
```

__Encountered problem__
The breaking changes in onsen-ui@2.2.0 affect the react-onsen-ui package  (mainly css component renaming). Are you considering to update the react-onsenui package somewhere in the near future. Would be nice to know your plans with this package. Thank you.

**Comment by frandiox** [comment]
@GoaGit `react-onsenui@1.2.0` was released the same day that `onsenui@2.2.0` was released. Have you tried to update `react-onsenui`?

**Comment by GoaGit** [comment]
My bad @frandiox. I mistakenly looked into the changelog of the react-onsenui repository only to realize later that the updates of the react package are now in the onsenui repository itself.
